docs/STEMMUS_SCOPE_BMI.md

```diff
@@ -0,0 +1,42 @@
+## STEMMUS_SCOPE BMI
+
+`STEMMUS_SCOPE` has a BMI-like interface. This is available when running the Matlab Runtime version of the model.
+The full BMI is implemented in Python in [PyStemmusScope](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/).
+
+A Docker image will also be available, which allows you to run the model on any system without installing Matlab Runtime.
+
+### BMI mode
+
+When starting the executable, a run-mode can be specified. The following command will run the model using the config file:
+
+```sh
+./STEMMUS_SCOPE "/home/path/to/config/file.txt" full
+```
+
+(Where `./STEMMUS_SCOPE` is the path to the executable. For more info see)
+
+To start BMI mode, pass anything (e.g. an empty string "") as config file, and use `bmi` to start the model in BMI-mode:
+
+```sh
+./STEMMUS_SCOPE "" bmi
+```
+
+The model will respond with `Finished command. Select run mode: `. Now you can initialize the mode:
+
+```
+Finished command. Select run mode: interactive "/home/path/to/config/file.txt"
```

Comment:
Above, it is mentioned, "Now you can initialize the model". However, here the command uses `interactive` instead of `initialize`. Can we show a command using `initialize` instead?

Reply:
Oh yeah I messed that up. It is indeed "initialize". I have also fixed the comment here on Github.